src/MCPClient/install/README.md

```diff
@@ -1,13 +1,184 @@
-## Configuration files
+## Introduction
 
-This directory contains the following files:
+MCPClient has multiple sources of configuration. This is the full list ordered
+by precedence (with the last listed item winning prioritization):
 
-- [`clientConfig.conf`](./clientConfig.conf) - a sample configuration file that
-the user can place in `/etc/archivematica/clientConfig.conf` to tweak the
-configuration of MCPClient.
+- Application defaults
+- Configuration files: (see [`the example`](./clientConfig.conf))
+    - `/etc/archivematica/archivematicaCommon/dbsettings` (optional)
+    - `/etc/archivematica/MCPClient/clientConfig.conf` (optional)
+- Environment variables (always win precedence)
 
-- [`clientConfig.logging.json`](./clientConfig.logging.json) - read the
-[logging configuration section](#logging-configuration) for more details.
+## Environment variables
+
+The value of an environment variable is a string of characters. The
+configuration system coerces the value to the types supported:
+
+- `string` (e.g. `"foobar"`)
+- `int` (e.g. `"60"`)
+- `float` (e.g. `"1.20"`)
+- `boolean` where truth values can be represented as follows:
+    - True (enabled):  `"1"`, `"yes"`, `"true"` or `"on"`
+    - False (disabled): `"0"`, `"no"`, `"false"` or `"off"`
+
+Certain environment strings are mandatory, i.e. they don't have defaults and
+the application will refuse to start if the user does not give them one.
+
+Please be aware that Archivematica supports different types of distributions
+(Ubuntu/CentOS packages, Ansible or Docker images) and they may override some
+of these settings or provide values to mandatory fields.
+
+This is the full list of environment strings supported:
+
+- **`ARCHIVEMATICA_MCPCLIENT_MCPCLIENT_SHAREDDIRECTORYMOUNTED`**:
```

Comment:
I'm not sure wich order follows the list. 

What do you think about listing first the mandatory options, and all the others next, in alphabetic order?

Reply:
I'll move the mandatory options to the top. If you don't mind I'll ignore the fact that the order is kind of random for now and leave it as a future improvement.